README.md

```diff
@@ -1,40 +1,44 @@
-`dpack`
-===
+# [`dpack`](https://github.com/dapphub/dpack)
 
-<center>
-  <img alt="a picture of dpack-1, the friendly mascot for the dpack-1 format"
-       src="https://x.mypinata.cloud/ipfs/QmZ9RVSRkTsqVg4CHdHfQQxWXmHWrWwoF3Nd2t5E6cz6fN"
-  ></img>
-</center>
+<img alt="a picture of dpack-1, the friendly mascot for the dpack-1 format" align="right"         src="https://x.mypinata.cloud/ipfs/QmZ9RVSRkTsqVg4CHdHfQQxWXmHWrWwoF3Nd2t5E6cz6fN" height="320"></img>
 
-A dpack is a file with a collection of EVM addresses and artifacts (ABIs).
+> `dpack` is a file with a collection of:
+>
+> - EVM addresses
+> - compiled artifacts (ABIs)
 
-The most important part of dpack is the `dpack-1` file format.
+The most _important_ part of dpack is the `dpack-1` file format.
 It has a simple and final spec that one person can learn in one sitting.
 New features will be deferred until the next version, which will be backwards
 compatible.
 
-This repo is a lightweight javascript package which lets you
+This repo is a lightweight JavaScript package which lets you
+
 - assemble dpacks from your deployment info
 - easily load and instantiate bindings for dapps
 
-For the time being, we are distributing dpack via npm:
+## Quickstart
+
+For the time being, we are distributing `dpack` via npm:
 
-`npm i @etherpacks/dpack  # WAIT! Read the next paragraph!`
+```shell
+npm i @etherpacks/dpack # WAIT! Read the next paragraph!
+```
 
-**WARNING**: Please take a minute to reflect on the nature of the software supply chain.
-What good is dpack if you are loading it via npm? Why not just stick the info in a package.json?
-There is no 'partially secured', we are either bootstrapped into a secure software supply chain, or not.
-If you are using `npm` as currently architected, you are not bootstrapped. But we have to start somewhere,
-so here we are.
+> **WARNING**: Please take a minute to reflect on the nature of the software supply chain.
+> What good is dpack if you are loading it via npm? Why not just stick the info in a `package.json`?
+> There is no 'partially secured', we are either bootstrapped into a secure software supply chain, or not.
+> If you are using `npm` as currently architected, you are not bootstrapped. But we have to start somewhere,
+> so here we are.
+> <sub><i>- nikolai mushegia; on dpack motivation, see discussion</i></sub>
```

Comment:
this line feels weird to merge lol like I'm talking in 3rd person

Reply:
do you not want attribution for the quote? The context otherwise is vague as to whom is addressing the reader.

Reply:
also misspelled your last name gg

Reply:
Well the entire document has the property that the authors are not explicitly mentioned, that is fine

> do you not want attribution for the quote? The context otherwise is vague as to whom is addressing the reader.